code/game/objects/items/implants/implant_deathrattle.dm

```diff
@@ -10,6 +14,20 @@
 		// use for making custom linked groups.
 		src.name = "[rand(100,999)] [pick(GLOB.phonetic_alphabet)]"
 
+/datum/deathrattle_group/offsite
+	area_whitelist = list(
+		/area/station,
+		/area/mine,
+	)
```

Comment:
I think this should include deepspace/ruins/gateways as well, specifically narrow it down to lavaland